debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlConnectorIT.java

```diff
@@ -194,6 +216,12 @@ public void shouldValidateAcceptableConfiguration() {
                                             .with(KafkaDatabaseHistory.TOPIC, "my.db.history.topic")
                                             .with(MySqlConnectorConfig.INCLUDE_SCHEMA_CHANGES, true)
                                             .build();
+        try {
+            Thread.sleep(5000);
```

Comment:
This will always sleep for 5 seconds. Is there any way for this to wake up or poll for readiness?